src/components/MultiHopTrade/components/LimitOrder/types.ts

```diff
@@ -3,10 +3,10 @@ import type { Order } from '@shapeshiftoss/types'
 
 export enum LimitOrderRoutePaths {
   Input = '/limit',
-  Confirm = '/limit/confirm',
-  AllowanceApproval = '/limit/allowance-approval',
-  PlaceOrder = '/limit/place-order',
-  Orders = '/limit/orders',
+  Confirm = 'confirm',
```

Comment:
subroutes things - the previous version with absolute paths *should* work but there's most likely something with our routes abstraction in `<RoutesCommon />` / `BrowserRouterProvider` that makes it not.
This PR is *not* intended to be a full refactor of routes (despite it looking like it, 90% of the PR is making things match new react-router v6 `navigate` vs. `history.push()` so not going to look into this as this is already a monster, and realistically may never take a look, as relative subroutes do work fine, although annoying.